services/slam/fake/slam_test.go

```diff
@@ -50,6 +80,12 @@ func TestFakeSLAMStateful(t *testing.T) {
 		extra := map[string]interface{}{}
 		verifyGetMapStateful(t, rdkutils.MimeTypePCD, slamSvc, extra)
 	})
+
+	t.Run("Test getting a PCD map via streaming APIs advances the test data", func(t *testing.T) {
```

Comment:
[q] why isn't this addition in `TestFakeSLAMGetInternalStateStream` as this function will be removed soon

Reply:
Sorry I don't understand. Can you please explain?

Reply:
Why isn't this stateful test being down in `TestFakeSLAMGetInternalStateStream`? I may not be fully understanding the `verifyGetPointCloudMapStreamStateful` call but since the GetInternalState test will be removed soon this addition won't exist in the long term. If I am misunderstanding please let me know

Reply:
The name of this test is `TestFakeSLAMStateful`. `Stateful` here doesn't refer to the `GetInternalState` function. It refers to the behavior of the fake slam service over many calls. Basically, this tests the fact that every time you ask for a map, all datasets from all other APIs advance by one. 

Is there a better name I could give `TestFakeSLAMStateful` to make that intention more clear?